src/common-lib/error-reporter/errorReporter.ts

```diff
@@ -217,8 +219,10 @@ const errorReporter = (
       }
 
       if (isBrowser) {
-        const bugsnagAllowed = getHasConsentedTo("bugsnag");
-        if (!bugsnagAllowed) {
```

Comment:
I think this might have been broken. It looks like the expectation here was for a boolean but this function always returned the string `enabled` | `disabled` | `pending`